README.md

```diff
@@ -203,6 +203,36 @@ Example:
 
```

Comment:
- [ ] Kindly make sure you add working of the `Live Demo` as the current one is broken. It shows a `404` error page once clicked. I suggest you deploy your project online an use that link to the live demo. For deploying you can use **Github pages**, **[Netlify](https://app.netlify.com/signup/start)**, [Cloudflare](https://dash.cloudflare.com/ae210f32ca2d8599db9e917808e0caf9),...